org.worldpossible.ScriptLauncher.yml

```diff
@@ -0,0 +1,35 @@
+app-id: org.worldpossible.ScriptLauncher
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+branch: eos3
```

Comment:
On Flathub all apps have branch `stable`. It's conventional to just not specify it:

```suggestion
```